python-checks/src/main/java/org/sonar/python/checks/cdk/AbstractCdkResourceCheck.java

```diff
@@ -171,4 +176,42 @@ protected static Predicate<Expression> isFqn(String fqnValue) {
       .isPresent();
   }
 
+  protected static Predicate<Expression> isString(String str) {
+    return expression ->  Optional.of(expression)
+      .filter(expr -> expr.is(Tree.Kind.STRING_LITERAL)).map(StringLiteral.class::cast)
+      .filter(stringLiteral -> stringLiteral.trimmedQuotesValue().equals(str))
+      .isPresent();
+  }
```

Comment:
Duplicates `org.sonar.python.checks.cdk.ClearTextProtocolsCheckPart#isStringValue`

Reply:
In that case I guess the best is to remove this method isString and move the method isStringValue from ClearTextProtocolsCheckPart to AbstractCdkResourceCheck, so that we can use it anywhere, right ?

Reply:
Yes